core/capabilities.go

```diff
@@ -114,6 +115,7 @@ var CapabilityNameLookup = map[Capability]string{
 	Capability_ImageToVideo:               "Image to video",
 	Capability_Upscale:                    "Upscale",
 	Capability_AudioToText:                "Audio to text",
```

Comment:
@pschroedl For now, I think it's perfectly fine to add this as a hardcoded capability 👍🏻. In the future, we might want to explore making this mechanism more generic, but let's save that challenge for another time 😅.